lib/Notification/Hooks.php

```diff
@@ -98,14 +98,15 @@ public function generateInvitation(Room $room, array $participants) {
 	 * @param Room $room
 	 */
 	public function generateCallNotifications(Room $room) {
-		if ($room->getActiveSince() instanceof \DateTime) {
+		if (count($room->getUserIds(true)) > 0) {
```

Comment:
Could you please change this to use a query that returns the number of users directly? With the current query, the whole list of user ids needs to be generated which has an unnecessary memory / runtime overhead.

Reply:
This is the only thing I did a bit differently than you requested. If you think that I should use COUNT instead of the workaround I did, please let me know.

Reply:
Yes, please use `COUNT(*)` instead.
See https://github.com/nextcloud/spreed/blob/2abbf31ced1252558edb90998102b19435d891ed/lib/Room.php#L775 for an example.

The function parameters could be
```php
public function getUserIds($inCall = false, $returnCount = false)
```